### PR TITLE
Add age

### PR DESCRIPTION
**Pull request by colinfrankb**

### CODE REVIEW

File: kagiso_auth/models.py [hunk 1]
```diff
@@ -48,6 +48,20 @@ def username(self):
...
+    def age(self):
+        if self.profile and 'birth_date' in self.profile:
+            birth_date = parser.parse(self.profile['birth_date'])
+
+            today = timezone.now()
+            birthday_has_passed = (today.month, today.day) < (birth_date.month, birth_date.day)  # noqa
+
+            # considering that int(True) is 1 and int(False) is 0
+            return today.year - birth_date.year - birthday_has_passed
+
```
Comment (MichaelVorster) [inline kagiso_auth/models.py]: Would it not be simpler to use something like 'datediff' and then just taking the year part?

Comment (colinfrankb) [inline kagiso_auth/models.py]: Just taking the year part does not take into consideration if the birthday is still upcoming or passed

File: kagiso_auth/tests/unit/test_models.py [hunk 2]
```diff
@@ -520,3 +521,41 @@ def test_record_sign_out_invalid_status_code_raises(self):
...
+
+    def test_age_returns_none_when_profile_is_none(self):
+        user = KagisoUser(
+            email='test@email.com',
+        )
+
+        assert user.age is None
+
+    def test_age_returns_none_when_birth_date_doesnt_exist(self):
+        user = KagisoUser(
```
Comment (MichaelVorster) [inline kagiso_auth/tests/unit/test_models.py]: This test is just a replication of the previous test, so I don't think it has any additional value.  I would rather give the user a profile, but without 'birth_date' present.

Comment (colinfrankb) [inline kagiso_auth/tests/unit/test_models.py]: You right, I mean't to actually add the profile attribute to the second test, as per my test name suggested, thanks for the spot

File: kagiso_auth/tests/unit/test_models.py [hunk 3]
```diff
@@ -520,3 +521,41 @@ def test_record_sign_out_invalid_status_code_raises(self):
...
+            email='test@email.com',
+        )
+
+        assert user.age is None
+
+    def test_age_returns_user_age_when_birthday_passed(self):
+        user = KagisoUser(
```
Comment (MichaelVorster) [inline kagiso_auth/tests/unit/test_models.py]: The names of these two last tests are not ideal.  Just from the names it seems that there is a function called 'age' that returns the age of a user. However comparing the names of these two tests, I get the impression that 'age' should either return or not return a user's age based on whether his birthday is upcoming (or passed).

File: kagiso_auth/models.py [hunk 4]
```diff
@@ -48,6 +48,20 @@ def username(self):
     def username(self, value):
         self.email = value
 
+        # http://stackoverflow.com/a/9754466
```
Comment (MichaelVorster) [inline kagiso_auth/models.py]: The reference to this stackoverflow issue appears twice.

Comment (colinfrankb) [inline kagiso_auth/models.py]: Thanks

File: kagiso_auth/tests/unit/test_models.py (hunk 3 above)
Comment (colinfrankb) [inline kagiso_auth/tests/unit/test_models.py]: What test name do you suggest then?

**Pull request merged**